adaptor-rm-webapp/src/main/java/com/sample/rm/RMToolManager.java

```diff
@@ -62,26 +77,17 @@
 
     private static final Logger log = LoggerFactory.getLogger(RMToolManager.class);
 
+    //TODO: Can we inject into a static???
+    @Inject static TrsEventHandler trsEventHandler;
```

Comment:
See `bindAsContract()` in https://stackoverflow.com/questions/49327986/getting-hk2-and-jersey-to-inject-classes